app/assets/stylesheets/components/tabs.scss

```diff
@@ -0,0 +1,136 @@
+@import '../config/import';
+
+.crayons-tabs {
+  // Setup
+  --border-width: 3px;
+  --tab-vertical-padding: var(--su-3);
+  --tab-horizontal-padding: var(--su-2);
+  --font-size: var(--fs-base);
+  display: flex;
+
+  @media (min-width: $breakpoint-s) {
+    --tab-vertical-padding: var(--su-2);
+  }
+
+  &--scrollable {
+    overflow-y: auto;
+    flex-wrap: nowrap;
+    -webkit-overflow-scrolling: touch;
+  }
+
+  &__item {
+    padding: var(--tab-vertical-padding) var(--tab-horizontal-padding);
+    display: inline-block;
+    text-align: center;
+    outline: none;
+    font-size: var(--font-size);
+    line-height: var(--su-6);
+    text-decoration: none;
+    cursor: pointer;
```

Comment:
Just curious about how you make the decision to use a CSS property vs. a utility class? In this case, i.e. `cursor-pointer` utility class vs. a CSS propert/value, `cursor: pointer`.

Reply:
Utility classes are more for scenarios when you want to either customize existing component or for example create w feature quickly without building the component itself. Or you need to build a particular view or piece of view and not sure how to handle that in frontend infra...

I've tried to give few scenarios for utility classes usage in one [comment for Vaidehi](https://github.com/thepracticaldev/dev.to/pull/7732#discussion_r421734732) - I hope that's helpful.